src/info.rs

```diff
@@ -138,12 +138,15 @@ fn pop_infobox(
             }
         }
         // fade out
-        color.0.set_a(lerp(
-            info_state.timer.elapsed_secs(),
-            0.,
-            info_state.timer.duration().as_secs_f32(),
-            1.,
-            0.,
-        ));
+        color
+            .0
+            .set(Box::new(lerp(
+                info_state.timer.elapsed_secs(),
+                0.,
+                info_state.timer.duration().as_secs_f32(),
+                1.,
+                0.,
+            )))
+            .unwrap();
```

Comment:
This unwrap causes the application to panic the application upon `SVG` exporting or when drag and dropping a map to the application (maybe on more events).

```
thread 'main' panicked at src/info.rs:150:14:
called `Result::unwrap()` on an `Err` value: 0.99783385
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
Encountered a panic in system `shu::info::pop_infobox`!
Encountered a panic in system `bevy_app::main_schedule::Main::run_main`!
```

(I am testing on `--release`.)